pkg/testing/integration_test.go

```diff
@@ -419,13 +419,19 @@ func (s *IntegrationSuite) TearDownSuite(c *C) {
 	// Uninstall app
 	if !s.skip {
 		err := s.app.Uninstall(ctx)
-		c.Assert(err, IsNil)
+		if err != nil {
+			c.Log("Failed to uninstall app %s: %v", s.name, err.Error())
+		}
+
 	}
 
 	// Uninstall implementation of the apps doesn't delete namespace
 	// Delete the namespace separately
 	err := s.cli.CoreV1().Namespaces().Delete(ctx, s.namespace, metav1.DeleteOptions{})
 	c.Assert(err, IsNil)
+	if err != nil {
+		c.Log("Failed to delete namespace: %v", err.Error())
```

Comment:
Will this execute after the Assert above?

What is the desired behavior?
* Is it to avoid stopping execution? then in that case `c.Check()` may be more appropriate and there is no need for an explicit `c.Log()` since `c.Check()` will do _"the right thing"_ ™️ 
* Not causing a test (teardown) failure on error? then remove the prior `c.Assert()`